content/en/users/aai/check-in/vos/authorisation/_index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Authorisation using VO information
+linkTitle: Authorisation
+type: docs
+weight: 10
+aliases:
+  - /users/check-in/vos/authorisation
+description: >
+  Authorisation using VO information
+---
+
+**Authorisation** is the mechanism handling permissions and access control.
+While authentication deals with the identification of a user, authorisation
+determines and enforces what a user can do in the system.
+
+This is depicted in the figure below, where a user access an application and
```

Comment:
Better practice would be to refer to a specific anchor, since layout may change.